kong/values.yaml

```diff
@@ -1,13 +1,15 @@
 Component: kong
 Name: kong
 Image: kong:0.13.0
+Registry: docker.io
 ImagePullPolicy: IfNotPresent
+ImagePullSecrets: []
 KongDatabase: postgres
 PostgresHost: kong-postgres
 HostPort: false
 IstioSidecar:
   enabled: false
-  hub: gcr.io/istio-release
+  hub: docker.io/istio-release
```

Comment:
Why are you changing the registry for this image?

Reply:
This is done to conform with all the charts we currently use.

Docker recommends using `docker.io`

Also this is now removed.